src/site/content/en/blog/browser-nativefs/index.md

```diff
@@ -164,19 +166,19 @@ const saveFile = async (blob) => {
 
 ## Introducing browser-nativefs
 
-As perfectly fine as the Native File System API is,
+As perfectly fine as the File System Access API is,
 it's [not yet  widely available](https://caniuse.com/#feat=native-filesystem-api).
```

Comment:
You may want to update caniuse data as well.

Reply:
Yes, I thought about this. Will do.